PR.md

```diff
@@ -0,0 +1,50 @@
+# Summary
+- **Status**: `draft` <!-- `review`, `suspended` -->
+- **RFCs**: [Link to RFC](#./link/to/rfc), [Link to RFC](#./link/to/rfc), or $\emptyset$.
+- **Categories**: any of `protocol-units`, `networks`, `scripts`, `util`, `cicd`, or `misc`.
+
+<!--
+Add your summary text here. 
+ -->
+
+# Core Software Components
+
+<!--
+Describe core software components added or modified by the PR. 
+-->
+
+# Ancillary Software Components
+<!--
+Describe components added or modified while working on core software components.
+-->
```

Comment:
I've seen many Repos where `PR.md` guidelines get ignored because their not relevant for the change. For small PRs there may be no changes to any Core or Ancillary software components, so these sections would be ignored. 

How about a more general section instead, just a # Changelog section where updates can be listed. 

If the developer wants to expand that and break it into sub-sections they can do so ad-hoc and we encourage their literary inclinations!

Reply:
I also like # Changelog because it can encapsulate all changes, for eg. updating docs, or a utility script, this wouldn't be a software component change.